test/tests/p5.Gain.js

```diff
@@ -0,0 +1,51 @@
+'use strict '
+
+define(['chai'],(chai)=>{
+    const expect = chai.expect ;
+    describe('P5.Gain',()=>{
+        it('can be initilized and disposed',()=>{
+            let gain = new p5.Gain();
+            gain.dispose();
+        });
+        it('can set input', ()=>{
+            let inputNode = new p5.Gain();
+            let mainGainNode  = new p5.Gain();
+
+            mainGainNode.setInput(inputNode);
+        });
+        it('can send output',()=>{
```

Comment:
this test is doing the same thing as 'can set input' maybe they can just be one test?

Reply:
though they appear to do the same task but actually we are able to validate twoo methods here 
setInput() and Connect() method

Reply:
ahh, got it! Thank you for clarifying and sorry I missed this